intranet/static/js/about.js

```diff
@@ -32,4 +31,4 @@ $(function() {
             mobileIndex++;
         }
     }, 5000);
-});
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2769584637/issues/source?bid=3176160&fileBranchId=3355598#l34)